### PR TITLE
[`Docs`] Fix typos

### PR DESCRIPTION
**Pull request by standardAI**
This pull request adds a copyright notice to relevant files and fixes some typos.
@stevhliu

### CODE REVIEW

File: docs/source/en/using-diffusers/sdxl_turbo.md [hunk 1]
```diff
@@ -49,11 +49,13 @@ pipeline = StableDiffusionXLPipeline.from_single_file(
 pipeline = pipeline.to("cuda")
 ```
 
+In this method, you need to set `timestep_spacing` parameter of the `StableDiffusionXLPipeline`'s scheduler to `'trailing'`.
```
Comment (stevhliu) [inline docs/source/en/using-diffusers/sdxl_turbo.md:52]: Can you modify the code snippet to show this?

**Review by stevhliu — COMMENTED** [review]
Thanks for the fixes!

File: docs/source/en/using-diffusers/sdxl_turbo.md (hunk 1 above)
Comment (standardAI) [reply docs/source/en/using-diffusers/sdxl_turbo.md:52]: Still being discussed here: #6112. Let's wait for what they say.

Comment (stevhliu) [reply docs/source/en/using-diffusers/sdxl_turbo.md:52]: I think something like this should work no? Then also mention that users can play around with the scheduler config values, but the main one is to set `timestep_spacing="trailing"`.

```py
from diffusers import StableDiffusionXLPipeline, EulerAncestralDiscreteScheduler
import torch
pipeline = StableDiffusionXLPipeline.from_single_file(
    "https://huggingface.co/stabilityai/sdxl-turbo/blob/main/sd_xl_turbo_1.0_fp16.safetensors", torch_dtype=torch.float16)
pipeline.scheduler = EulerAncestralDiscreteScheduler.from_config(pipeline.scheduler.config, timestep_spacing="trailing")

prompt = "A croissant shaped like a cute bear."

image = pipeline(prompt=prompt, guidance_scale=0.0, num_inference_steps=1).images[0]
image
```

File: docs/source/en/using-diffusers/sdxl_turbo.md [hunk 2]
```diff
@@ -41,19 +41,23 @@ pipeline = pipeline.to("cuda")
...
 
 pipeline = StableDiffusionXLPipeline.from_single_file(
-    "https://huggingface.co/stabilityai/sdxl-turbo/blob/main/sd_xl_turbo_1.0_fp16.safetensors", torch_dtype=torch.float16)
+    "https://huggingface.co/stabilityai/sdxl-turbo/blob/main/sd_xl_turbo_1.0_fp16.safetensors",
+    torch_dtype=torch.float16, variant="fp16")
 pipeline = pipeline.to("cuda")
+pipeline.scheduler = EulerAncestralDiscreteScheduler.from_config(pipeline.scheduler.config, timestep_spacing="trailing")
 ```
 
+When using [`~StableDiffusionXLPipeline.from_single_file`], users can play around with the scheduler config values, but the main one is to set `timestep_spacing="trailing"`.
```
Comment (stevhliu) [inline docs/source/en/using-diffusers/sdxl_turbo.md:54]: Let's put this right before the code example so users fully know what to expect.

```suggestion
For this loading method, you need to set `timestep_spacing="trailing"` (feel free to experiment with the other scheduler config values to get better results).
```

**Review by stevhliu — APPROVED** [review]
Thanks!

**Review by stevhliu — APPROVED** [review]

**Comment by standardAI** [comment]
Thanks for giving feedback and merging!
